### ISSUE TITLE
kiss version 5.5.21 and later detects circular dependencies when trying to remove seemingly any package  

### ISSUE THREAD

**Issue by trunc88** [gkisslinux/grepo#16]
starting with kiss version 5.5.21, I am unable to remove packages, even when setting KISS_FORCE=1. Posting here in the GKISS repo because the circular dependency seems to always involve glibc.

`$ kiss remove weechat`
`ERROR Circular dependency detected perl <> openssl`

`$ kiss depends openssl`
`perl make`

`$ kiss depends perl`
`bzip2`
`glibc`
`zlib`

`$ kiss depends glibc`
`gawk make`
`linux-headers make`
`python make`

`$ kiss depends python`
`bzip2`
`expat`
`libffi`
`ncurses`
`openssl`
`readline`
`sqlite`
`xz`
`zlib`

**Comment by git-bruh** [comment]
This was fixed https://github.com/kisslinux/kiss/issues/225, try rebuilding everything that depends on glibc so the dependency is removed and also updating to the latest kiss

`kiss revdep glibc | sed 's|/.*||g' | xargs kiss b`

**Comment by trunc88** [comment]
Oh that's right, I saw that commit when I was looking for what changed. I thought I had tried rebuilding the weechat-related packages that depended on glibc, but obviously I forgot to do perl, even though I posted its dependencies right there. Woops! Rebuilding the glibc-dependent packages does fix the circular dependencies. Now trying to rebuild all glibc-dependent packages, I am running into an error building glib. [Build log here](https://clbin.com/BJgUY). Maybe this belongs in a new issue at the original KISS repo.
